src/test/shell/bazel/cc_integration_test.sh

```diff
@@ -1708,4 +1708,48 @@ EOF
   expect_log "in pkg/library.cpp: ''"
 }
 
+function test_compiler_flag_gcc() {
```

Comment:
Looks like your other diff snuck in here

Reply:
It's stacked as the other PR hasn't been merged yet.

Reply:
I rebased onto master, so this is no longer stacked.